code/download_resource.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Download the resources specified on the command line
+
+
+# download a resource from a URL (assumed to be geoserver) to a CSV file
+# $1 - full path to resource file
+# $2 - directory to save the CSV file into
```

Comment:
Hey @mhidas, I don't have much experience in bash scripting, so this may be more of a question rather than a suggestion. Is there any reason for:
``` 
# $1 - full path to resource file
# $2 - directory to save the CSV file into 
```
Considering the usage is `download_resource.sh output_dir resource_file` would not be clearer to have: 
```
# $1 - directory to save the CSV file into
# $2 - full path to resource file
```

Reply:
Ah, right, I agree it's a bit confusing. The "usage" message is for the `download_resource.sh` script you run on the command line, while the above comment is documenting the arguments to the `extract_resource` function, which is only used internally. However, there's no reason the two couldn't be ordered the same way, so I'll switch them around.

Reply:
Fixed in last commit